llvm/test/CodeGen/AMDGPU/set-inactive-wwm-overwrite.ll

```diff
@@ -13,7 +13,6 @@ define amdgpu_cs void @if_then(ptr addrspace(8) inreg %input, ptr addrspace(8) i
 ; GCN-NEXT:    s_or_b32 exec_lo, exec_lo, s0
 ; GCN-NEXT:    v_cmp_lt_u32_e32 vcc_lo, 3, v0
 ; GCN-NEXT:    s_and_saveexec_b32 s0, vcc_lo
-; GCN-NEXT:    s_cbranch_execz .LBB0_4
```

Comment:
I'm surprised this branch is now deleted. I thought the cutoff for profitability would be somewhere around 6 instructions, but this block has 9 + the branch